README.md

```diff
@@ -71,29 +82,52 @@ isIPFS.ipfsPath('/ipfs/invalid-hash') // false
 
 isIPFS.ipnsPath('/ipfs/QmYjtig7VJQ6XsnUjqqJvj7QaMcCAwtrgNdahSiFofrE7o') // false
 isIPFS.ipnsPath('/ipns/github.com') // true
+
+isIPFS.subdomain('http://bafybeiabc2xofh6tdi6vutusorpumwcikw3hf3st4ecjugo6j52f6xwc6q.ipns.dweb.link') // true
```

Comment:
Is this true for `http://bafybeie5gq4jxvzmsym6hjlwxej4rwdoxt7wadqvmmwbqi7r27fclha2va.ipfs.dweb.link` also? Maybe add an example here :D